.github/workflows/sync-main-to-preprod-and-prod.yml

```diff
@@ -102,6 +88,22 @@ jobs:
           # TODO this does not check whether the updates are in main or in prod, only that files differ. We could
           # make this more exact later to check that the updates are in main (anything else is unexpected).
           diff -qr repository/repository sigstore-tuf-root | grep -Po '([0-9\.]*\w+[\.\w+]*(?= differ))|((Only in \w+\: )\K(.*))' | while read l; do check_filename $l; done
+      - name: sync
+        id: sync
+        if: ${{ ! steps.check-updated-files.outputs.abort }}
```

Comment:
I always forget the evaluation rules for conditional step execution. If `abort=false` is added to the previous step, would this still execute, or will the test only eval to `true` if `abort == true`?

Reply:
outputs are strings so I think "false" (if it was used) and "true" would lead to same result here.

Reply:
Yeah, that's what I was feared, so we probably need to rewrite to `steps.check-updated-files.outputs.abort = true`.

Reply:
Good catch, thanks.